aggregator_api/src/lib.rs

```diff
@@ -113,13 +115,23 @@ async fn auth_check(conn: &mut Conn, State(cfg): State<Arc<Config>>) -> impl Han
         constant_time::verify_slices_are_equal(bearer_token.as_ref(), key.as_ref()).is_ok()
     }) {
         // Authorization succeeds.
+        conn.set_state(cfg);
         return None;
     }
 
     // Authorization fails.
     Some((Status::Unauthorized, Halt))
 }
 
+async fn get_config(_: &mut Conn, State(config): State<Arc<Config>>) -> Json<AggregatorApiConfig> {
+    Json(AggregatorApiConfig {
+        dap_url: config.public_dap_url.clone(),
+        role: AggregatorRole::Either,
+        vdafs: vec![1, 2, 3],
+        query_types: vec![1, 2],
```

Comment:
I believe we can reuse `messages::query_type::Code` in `messages/src/lib.rs` for this (somewhat unfortunately named):
```
    /// DAP protocol message representing the type of a query.
    #[derive(Copy, Clone, Debug, PartialEq, Eq, TryFromPrimitive, Serialize, Deserialize)]
    #[repr(u8)]
    pub enum Code {
        Reserved = 0,
        TimeInterval = 1,
        FixedSize = 2,
    }
```

Or I would define a model in `mod models`.

Reply:
addressed in 05026e5e02500027dfc843bc0710789b4c8d595a